wire-ios-data-model/Tests/Source/ManagedObjectContext/DatabaseMigrationTests.swift

```diff
@@ -34,6 +34,21 @@ final class DatabaseMigrationTests: DatabaseBaseTest {
         "handle"
     ]
 
+    func testEventsLatestModelHasMigrationVersion() throws {
+        // given
+        let latestMigrationVersion = CoreDataEventsMigrationVersion.allCases.first
```

Comment:
@samwyndham is this the kind of test you were looking for ?

Reply:
Looks good!